Component Plug-in (CP) Examples/mapField/mapField/v1/index.html

```diff
@@ -39,22 +42,34 @@
     });
 
     function initMap() {
+      Appian.Component.setValidations('');
       mapState.scriptLoaded = true;
-      changeLocation(newLocation)
+      changeLocation(newLocation);
     }
 
+    function gm_authFailure() {
+      Appian.Component.setValidations('The API Key provided is invalid. Please update the parameter "Google API Key" with a valid key.');
+    };
+
     function changeLocation(newLocation) {
-      currentLocation = newLocation;
-      // Obtain coordinates for user input location
-      mapState.geocoder = mapState.geocoder || new google.maps.Geocoder();
-      mapState.geocoder.geocode({ 'address': newLocation }, function (results, status) {
-        let latLng = results[0].geometry.location;
-        if (mapState.map) {
-          mapState.map.setCenter(latLng);
-        } else {
-          renderMap(latLng);
-        }
-      });
+      if (newLocation) {
+        currentLocation = newLocation;
+        // Obtain coordinates for user input location
+        mapState.geocoder = mapState.geocoder || new google.maps.Geocoder();
+        mapState.geocoder.geocode({ 'address': newLocation }, function (results, status) {
+          if (status === 'OK') {
+            Appian.Component.setValidations('');
```

Comment:
May want to add a comment above this like `// Clear out any existing validations`